huaweicloud/resource_huaweicloud_compute_volume_attach_test.go

```diff
@@ -121,18 +121,6 @@ func testAccCheckComputeVolumeAttachExists(n string, va *block_devices.VolumeAtt
 	}
 }
 
-func testAccCheckComputeVolumeAttachDevice(
```

Comment:
suggest to keep this function,we will refactor later.

Reply:
pls add when you need really